python/py-mitmproxy/Portfile

```diff
@@ -38,7 +38,8 @@ python.versions     37 38 39
 if {${name} ne ${subport}} {
     depends_build-append    port:py${python.version}-setuptools
 
-    depends_lib-append      port:py${python.version}-asn1 \
+    depends_lib-append      port:py${python.version}-asgiref \
```

Comment:
`py-asgiref` has only subports for PY38/PY39, so you'll need to add a PY37 subport there as well to get `py37-mitmproxy` to build. I am not sure why `port lint` doesn't warn about this...